google/cloud/security/scanner/samples/group_rules.yaml

```diff
@@ -1,50 +1,21 @@
-rules:
-  # Only allow Organization users to be in the Organization groups.
-  - name: sample whitelist for group users
-    mode: whitelist
-    resource:
-      - type: group
-        applies_to: self_and_children
-        resource_ids:
-          - my_customer
-    inherit_from_parents: true
-    members:
-      - role: MEMBER
-        type: USER
-        email: '*@YOUR_ORG_EMAIL_DOMAIN'
+- name: Allow my company users and gmail users to be in my company groups.
+  group_email: my_customer
+  mode: whitelist
+  conditions:
```

Comment:
Nit: I know what conditions means because we discussed this offline already, but for someone new to the scanner, it may not be clear what exactly is the condition (i.e. a condition implies something like an "if" statement, you're checking if something eq/ne/lte/gte etc.)

Suggestion is to use some kind of stronger indicator (or property name) to imply that you're checking equality